src/BlazorWebView/src/SharedSource/WebView2WebViewManager.cs

```diff
@@ -177,27 +177,46 @@ protected override void NavigateCore(Uri absoluteUri)
 		{
 			_ = Dispatcher.InvokeAsync(async () =>
 			{
-				await _webviewReadyTask;
-				_webview.Source = absoluteUri;
+				var isWebviewInitialized = await _webviewReadyTask;
+
+				if (isWebviewInitialized)
+				{
+					_webview.Source = absoluteUri;
+				}
 			});
 		}
 
 		/// <inheritdoc />
 		protected override void SendMessage(string message)
 			=> _webview.CoreWebView2.PostWebMessageAsString(message);
 
-		private async Task InitializeWebView2()
+		private async Task<bool> TryInitializeWebView2()
 		{
 			var args = new BlazorWebViewInitializingEventArgs();
 #if WEBVIEW2_MAUI
 			_blazorWebViewHandler.VirtualView.BlazorWebViewInitializing(args);
-			_coreWebView2Environment = await CoreWebView2Environment.CreateWithOptionsAsync(
-				browserExecutableFolder: args.BrowserExecutableFolder,
-				userDataFolder: args.UserDataFolder,
-				options: args.EnvironmentOptions)
-				.AsTask()
-				.ConfigureAwait(true);
-			await _webview.EnsureCoreWebView2Async();
+
+			try
+			{
+				_coreWebView2Environment = await CoreWebView2Environment.CreateWithOptionsAsync(
+					browserExecutableFolder: args.BrowserExecutableFolder,
+					userDataFolder: args.UserDataFolder,
+					options: args.EnvironmentOptions)
+					.AsTask()
+					.ConfigureAwait(true);
```

Comment:
nit;

```suggestion
					options: args.EnvironmentOptions)
						.AsTask()
						.ConfigureAwait(true);
```
```suggestion
					options: args.EnvironmentOptions)
					.AsTask()
					.ConfigureAwait(true);
```

Reply:
These lines are already indented one extra level - does this suggestion follow our decided conventions? I'm not implying it doesn't, but this would be different than how the formatting was before the change (and how the formatting is in a syntactically similar location later in the method).